lib/ts/recipe/dashboard/api/multitenancy/createOrUpdateThirdPartyConfig.ts

```diff
@@ -0,0 +1,68 @@
+/* Copyright (c) 2024, VRAI Labs and/or its affiliates. All rights reserved.
+ *
```

Comment:
need to make sure there are no calls to the core in a loop in any of the dashboard apis

Reply:
Done